demo/README.md

```diff
@@ -643,3 +644,28 @@ python demo/demo_video_structuralize.py
     --label-map-stdet tools/data/ava/label_map.txt \
     --label-map tools/data/kinetics/label_map_k400.txt
 ```
+
+## Audio Demo
+
+Demo script to predict the audio-based action recognition using a single audio feature. To extract audio features use the `build_audio_features.py` script.
```

Comment:
Maybe one more bullet can be added to tell users how to extract audios from a video, i.e., using the `extract_audio.py` script.